tools/deployment/package/docker-compose.runtime.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  clp-runtime:
+    hostname: "clp_runtime"
+    image: "${CLP_PACKAGE_CONTAINER_IMAGE_REF:-clp-package}"
+    logging:
+      driver: "local"
+    stdin_open: true
+    tty: true
+    user: "${CLP_FIRST_PARTY_SERVICE_UID_GID:-1000:999}"
+    environment:
+      HOME: "${HOME:?Please set a value.}"
+      CLP_HOME: "${CLP_HOME:?Please set a value.}"
+      CLP_PWD_HOST: "${CLP_PWD_HOST:-${PWD}}"
+    volumes:
```

Comment:
Shall we organize these into groups? E.g., Docker-related mounts and other mounts?